pytensor/xtensor/type.py

```diff
@@ -720,7 +830,7 @@ def transpose(
         *dim: str | EllipsisType,
         missing_dims: Literal["raise", "warn", "ignore"] = "raise",
     ):
-        """Transpose dimensions of the tensor.
+        """Transpose the dimensions of the variable.
 
         Parameters
         ----------
```

Comment:
In the description of missing dims, if you want the bullet point list to be rendered as a list instead of as inline text (see current render https://pytensor--1533.org.readthedocs.build/en/1533/library/xtensor/type.html#pytensor.xtensor.type.XTensorVariable.transpose) there should be an empty line between previous line and start of list:

```
How to handle dimensions that don't exist in the tensor:

- "raise": Raise an error if any dimensions don't exist
```

Reply:
Fixed, this was pre-existing